web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -425,11 +613,11 @@ Your app should be working correctly and using new auth, but to finish the migra
 
   The database migrations will automatically run on successful deployment of the server and delete the now redundant auth-related `User` columns from the database.
 
-  Your app is now fully migrated to the new auth system 🎉!
+  Your app is now fully migrated to the new auth system.
```

Comment:
```suggestion
  Your app is now fully migrated to the new auth system 🎉!
```

Reply:
In our docs writing guide we discourage the use of emojis: https://github.com/wasp-lang/wasp/blob/main/web/docs/writingguide.md#grammar